docs/BR.md

```diff
@@ -1008,6 +1043,62 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
+
+Results or information obtained from one Network Perspective MUST NOT be reused or cached when performing validation through subsequent Network Perspectives (e.g., different Network Perspectives cannot rely on a shared DNS cache to prevent an adversary with control of traffic from one Network Perspective from poisoning the DNS cache used by other Network Perspectives). The network infrastructure providing Internet connectivity to a Network Perspective MAY be administered by the same organization providing the computational services required to operate the Network Perspective.
+
+CAs MAY immediately retry Multi-Perspective Issuance Corroboration using the same validation method or an alternative method (e.g., a CA can immediately retry validation using "Email to DNS TXT Contact" if "Agreed-Upon Change to Website - ACME" does not corroborate the outcome of Multi-Perspective Issuance Corroboration). When retrying Multi-Perspective Issuance Corroboration, CAs MUST NOT rely on corroborations from previous attempts. There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
+
+The "Quorum Requirements" Table describes quorum requirements related to Multi-Perspective Issuance Corroboration. Network Perspectives are considered distinct when the straight-line distance between the two States, Provinces, or Countries they reside in is at least 500 km. Network Perspectives are considered "remote" when they are distinct from the Primary Network Perspective and the other Network Perspectives represented in a quorum.
+
+Table: Quorum Requirements
+
+| # of Distinct Remote Network Perspectives Used | # of Allowed non-Corroborations |
+| --- | --- |
+| 2-5 |  1  |
+| 6+  |  2  |
+
+Network Perspectives performing Multi-Perspective Issuance Corroboration:
+
+- MUST:
+    - Forward all Internet traffic via a network or set of networks that filter all RPKI-invalid BGP routes as defined by RFC 6811. 
+
+- SHOULD:
+    - Facility & Service Provider Requirements
+      - Be hosted from an ISO/IEC 27001 certified facility or equivalent security framework independently audited and certified or reported. 
+      - Rely on services covered in one of the following reports: System and Organization Controls 2 (SOC 2), IASE 3000, ENISA 715, FedRAMP Moderate, C5:2020, CSA STAR CCM, or equivalent services framework independently audited and certified or reported.
+    - Network Hardening
```

Comment:
Apologies if this is already handled, is outside the scope of this ballot, but should there be any requirement on how the CA communicates with the remote network perspective?

For example, is the remote network perspective was a simple DNS Resolver, and the CA queried it with a UDP DNS request. Would that meet the requirements of the ballot, but also still be susceptible to a MiM attacl

Reply:
That would make sense to me. I think there was always a bit of an understanding in the working group that communication between the different network perspectives needed to be over a secure and authenticated channel to avoid those types of attacks. I feel some ballot language to clarify that would be a good idea.

Reply:
Thanks @birgelee
Where would be the best place to add this language?
@ryancdickson, do you have thoughts on this?

Reply:
@kingcdavid - How do you feel about the following addition?

_"All communications between remote Network Perspectives and the CA MUST be encrypted using Transport Layer Security (TLS) Version 1.2 or higher using secure ciphersuites that ensure forward secrecy. TLS Version 1.3 SHOULD be used."_

Suggestions welcome.

Reply:
I think that looks good, Is there any alternative to TLS though? i.e. would un-encrypted traffic (DNS UDP again) over a IPSec VPN also be secure enough? Just thinking we may not want to restrict on technology?

instead we could have:

> "All communications between remote Network Perspectives and the CA MUST be encrypted using suitable encryption methods, i.e Transport Layer Security (TLS). If TLS, Version 1.2 or higher using secure ciphersuites that ensure forward secrecy. TLS Version 1.3 SHOULD be used."

Reply:
@kingcdavid I defiantly think not being specific to a particular authenticated encryption method is a good idea. I feel an IPSec VPN would give proper security. Another technique for communication with remote perspectives that I have heard discussed is using WireGuard which is also quite secure and a commonly used practice.

I can't tell if it makes sense to add something about the encryption being "authenticated" just to clarify that it should be robust even against an adversary that can manipulate packets (e.g., not just a plane DF key exchange that is vulnerable to an on-path adversary but would count as encryption).

Reply:
How about: "_All communications between remote Network Perspectives and the CA MUST take place over an authenticated and encrypted channel relying on modern protocols (e.g., over HTTPS)._" ?

Reply:
I'm good with that, ensures that it will be secure and also give scope to use any suitable technology

Thanks!

Reply:
Added here: https://github.com/ryancdickson/staging/commit/6bda42e6307b2c62c952df946c2144c17d13f55c

The existing "MUST" list didn't feel like an appropriate location given those requirements are specific to remote perspectives --- and this change also implicates the CA.

Reply:
Marking thread as resolved. Thanks for feedback @kingcdavid and @birgelee!